### DISCUSSION TITLE
Label Phone only Meeting as "phone" and not "oniine"

### DISCUSSION THREAD

**Discussion by WAIA12** [code4recovery/12-step-meeting-list#1236] [Q&A]
The Meeting Guide  shows phone only meetings with an icon of a phone.  We would like our 12th step meeting list to show phone meeting and not online as it does today.  The example group, Andrew's Armed Forces, has hybrid meetings using a phone conference phone number in the In Person meetings and at other times  they have  phone conference only meetings

If the Meeting Guide App can show phone only meetings, we assume the 12 step plugin should also be able to do it.  We have several other phone only meetings that would like there meetings correctly identified as phone meetings.

If you can provide us  with lines of code and where to put them, we have a couple of people who can go in and fix it. 

![AndrewsPlugInImage](https://github.com/code4recovery/12-step-meeting-list/assets/148124285/80c74363-aac6-4f21-905e-0b4aaa10c705)
![AndrewsMeetinGuideImage](https://github.com/code4recovery/12-step-meeting-list/assets/148124285/45aa5ba7-484e-41ed-b6b5-9d648edd1310)
  Thanks!
Luellla Tabor, WAIA Office Administrator, 202-966-9783

**Comment by joshreisner** [comment]
hi Luella! one option might be to switch to the TSML UI appearance? this is in WordPress Dashboard > Meetings > Settings > User Interface Display -- it will make the meeting finder look like https://aasanjose.org/meetings - this appearance makes the distinction between types of meetings

<img width="1218" alt="Screenshot 2023-10-16 at 10 54 32 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/b740df5a-eda4-427c-b021-0c93c7102a6f">

We've been investing more time in improving this UI than in the "Legacy" interface - let us know if that works for you

**Comment by WAIA12** [comment]
Josh,

Thanks for the quick response..  I'm not sure we're ready to change
formats.  You know how "people" hate change but that may be our best
option.

Luella

**Reply by joshreisner** [reply]
i hear you! there are number of other benefits besides the format: 

* meeting list starts at now, so the top meeting is the next one (even applies if the user is in a different time zone) this is helpful because it means the user (who is most likely using a mobile device) does not need to scroll down as far (see pictures below, or just compare aa-dc.org/meetings to aasanjose.org/meetings)
* there is a "meetings in progress" bar at the top
* add to calendar button
* works in user's native language automatically if device language is spanish, french, swedish, or japanese

maybe something to discuss in tech committee, or bring to intergroup?

TSML
![aa-dc org_meetings(iPhone 12 Pro)](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/389187c4-ac0e-41a8-86ef-87ea59d87c5e)

TSML UI
![aasanjose org_meetings(iPhone 12 Pro)](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/4d361ab3-8e2d-4d7d-aec6-f9253b30dd45)

**Comment by WAIA12** [comment]
Thanks!  I'll forward this to our Tech Committee.
